app/src/examples/WorkletRuntimeExample.tsx

```diff
@@ -122,6 +130,72 @@ function PerformanceNowDemo() {
   return <Button title="performance.now" onPress={handlePress} />;
 }
 
+function BackgroundQueueFromJSDemo() {
+  const handlePress = () => {
+    const runtime = createWorkletRuntime('foo');
+    const queue = createBackgroundQueue('bar');
+    runOnBackgroundQueue(queue, runtime, () => {
+      'worklet';
+      console.log('Hello from background!', Math.random());
+    })();
+  };
+
+  return <Button title="Background queue from JS" onPress={handlePress} />;
+}
+
+function BackgroundQueueFromUIDemo() {
+  const handlePress = () => {
+    const runtime = createWorkletRuntime('foo');
+    const queue = createBackgroundQueue('bar');
+    runOnUI(() => {
+      'worklet';
+      const x = Math.random();
+      console.log('Hello from UI thread!', x);
+      runOnBackgroundQueue(queue, runtime, () => {
```

Comment:
Yep, it's possible to call `runOnBackgroundQueue` from any worklet runtime (including the UI runtime) since it just calls `_scheduleOnBackgroundQueue` which is injected to all worklet runtimes.